### DISCUSSION TITLE
Anyone try the POE ESP32?

### DISCUSSION THREAD

**Discussion by stevewa2066** [markqvist/Reticulum#464] [General]
This could be useful. Wondering if anyone has tried it? Load the Rnode on it. 

https://hackaday.io/project/179601-lora-aprs-igate-on-esp32-poe-board

**Comment by markqvist** [comment]
I did see that board, and found it really interesting. Haven't actually tried messing around with it, but it should definitely be possible to create an RNode variant for it.

**Comment by stevewa2066** [comment]
I have the parts… guess I should start soldering .

Steve N0FPF

Pardon the brevity, sent from a mobile device.

**Reply by markqvist** [reply]
Seems like it's just using the already supported SX1278 chip internally, so with a little luck, it should be pretty easy to get it up and running.

There'll probably be some work in getting the host connection working over ethernet instead of serial/bluetooth, but just getting it running over the normal connections would probably be a good start.
